test/less/declaration/2.json

```diff
@@ -0,0 +1,81 @@
+{
+  "type": "declaration",
+  "content": [
+    {
+      "type": "property",
+      "content": [
+        {
+          "type": "ident",
+          "content": "@{prop}",
```

Comment:
I don't understand why not. That's how the parser is setup to handle that part. This fixes a pretty critical oversight and breaking bug in the 3.x versions that worked fine with 3.0.0.-31. At the very least, with all tests passing, it should be merged and "corrected" in a patch version.